src/style.css

```diff
@@ -0,0 +1,105 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
```

Comment:
Remove redundant default value

You can check your styles through devtools. If you have disabled a style and nothing has changed, then it is redundant and should be deleted
![image](https://user-images.githubusercontent.com/70009205/213420619-ea220821-2670-4c86-9832-2bf3b9c45e9d.png)